eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -16,25 +16,28 @@ parameters:
     Linux_Python35:
       OSVmImage: 'ubuntu-18.04'
       PythonVersion: '3.5'
+      CoverageArg: '--disablecov'
     MacOs_Python37:
       OSVmImage: 'macOS-10.15'
       PythonVersion: '3.7'
+      CoverageArg: '--disablecov'
     Windows_Python27:
       OSVmImage: 'windows-2019'
       PythonVersion: '2.7'
+      CoverageArg: '--disablecov'
     Linux_PyPy3:
       OSVmImage: 'ubuntu-18.04'
       PythonVersion: 'pypy3'
+      CoverageArg: '--disablecov'
     Linux_Python39:
       OSVmImage: 'ubuntu-18.04'
       PythonVersion: '3.9'
+      CoverageArg: ''
 
 jobs:
   - job: ${{ parameters.JobName }}
     variables:
       skipComponentGovernanceDetection: true
-      CoverageArg: --disablecov
```

Comment:
Why keep this as default and then only have the specific matrix entries override it with empty if the don't want to disable?